proto/interchain_security/ccv/provider/v1/query.proto

```diff
@@ -35,20 +35,36 @@ service Query {
         "/interchain_security/ccv/provider/consumer_chains";
   }
 
+  // [DEPRECATED]: Use `QueryConsumersThatAreAboutToStart` instead
   // QueryConsumerChainStarts queries consumer chain start proposals.
   rpc QueryConsumerChainStarts(QueryConsumerChainStartProposalsRequest)
       returns (QueryConsumerChainStartProposalsResponse) {
+    option deprecated = true;
```

Comment:
I deprecate the `QueryConsumerChainStarts` and `QueryConsumerChainStops` queries.
These queries seem to be returning consumers that are about to start or stop respectively. However, I do not not really like the naming of their arguments `...Proposals...`  because with permissionless we have chains that are about to start or stop outside a proposal. Because of this, I deprecate the queries in favor of `QueryConsumersThatAreAboutToStart` and `QueryConsumersThatAreAboutToStop`.

Reply:
Why deprecate and not just remove them?

Reply:
It won't be backward compatible if I just remove them. I don't know who might be using those queries and I wouldn't want to break their builds.

Reply:
As long as we add clear changelogs is fine to break compatibility, especially for queries.

Reply:
The cost of deprecating them seems almost 0. I do not see the benefit of removing them. But if you insist, I can go ahead and remove them. Let me know.

Reply:
As the logic changed. SDK does the same thing. It's fine to break some things to reduce code complexity.